dynamallow/model.py

```diff
@@ -129,9 +148,10 @@ class Table:
                 write = 1
 
             class Schema:
-                id = fields.String(required=True)
-                name = fields.String()
-                color = fields.String(validate=validate.OneOf(('purple', 'red', 'yellow')))
```

Comment:
How do validations work in Schematics?  Can you add an example of a "choice", like the old example.